app/jobs/whats_app_adapter/three_sixty_dialog/upload_file.rb

```diff
@@ -5,43 +5,41 @@
 module WhatsAppAdapter
   module ThreeSixtyDialog
     class UploadFile < ApplicationJob
-      # rubocop:disable Metrics/AbcSize
       def perform(message_id:)
         @message_id = message_id
         message = Message.find_by(id: message_id)
-        return unless message
 
         request = message.request
         organization = Organization.find_by(id: request.organization.id)
```

Comment:
So we let `NilPointerError` happen and let the job crash?

Reply:
Yes, I added these guard clauses to turn the jobs into no-ops in the case that the message doesn't exist, for example, it was deleted before the job actually ran. However, after discussing this with @soey we agreed that for our use case, it would be more helpful if it did fail loudly vs quietly so that we know there is a potential issue.

We could change line 11 to a simple find, which will through an error if there is no message with the id message_id, but I'm not sure it's _that_ important the specific error.